pkg/backend/httpstate/policypack.go

```diff
@@ -303,3 +321,57 @@ func installRequiredPolicy(finalDir string, tarball []byte) error {
 
 	return nil
 }
+
+func completePythonInstall(finalDir, projPath string, proj *workspace.PolicyPackProject) error {
+	// Create virtual environment.
+	fmt.Println("Creating virtual environment for policy pack...")
+	fmt.Println()
+
+	venvDir := filepath.Join(finalDir, "venv")
+	cmd, err := python.Command("-m", "venv", venvDir)
+	if err != nil {
+		return err
+	}
+
+	if output, err := cmd.CombinedOutput(); err != nil {
+		if len(output) > 0 {
+			os.Stdout.Write(output)
+			fmt.Println()
+		}
+		return errors.Wrapf(err, "creating virtual environment at %s", venvDir)
+	}
+
+	// Save project with venv info.
+	proj.Runtime.SetOption("virtualenv", "venv")
+	if err := proj.Save(projPath); err != nil {
+		return errors.Wrapf(err, "saving project at %s", projPath)
+	}
+
+	fmt.Println("Finished creating virtual environment")
+	fmt.Println()
+
+	requirementsPath := filepath.Join(finalDir, "requirements.txt")
+	if _, err := os.Stat(requirementsPath); os.IsNotExist(err) {
+		return nil
+	}
+
+	fmt.Println("Installing dependencies for policy pack...")
+	fmt.Println()
+
+	pipCmd := python.VirtualEnvCommand(venvDir, "pip", "install", "-r", "requirements.txt")
+	pipCmd.Dir = finalDir
+	pipCmd.Env = python.ActivateVirtualEnv(os.Environ(), venvDir)
+
+	if output, err := pipCmd.CombinedOutput(); err != nil {
+		if len(output) > 0 {
+			os.Stdout.Write(output)
+			fmt.Println()
+		}
+		return errors.Wrap(err, "installing dependencies via `pip install -r requirements.txt`")
+	}
+
+	fmt.Println("Finished installing dependencies")
+	fmt.Println()
```

Comment:
There's a fair bit of output here.  Not a big deal - but is the UX of this "good"?

Reply:
The UX isn't very good. The current output looks like:

```
Downloading policy pack
Unpacking policy pack "/Users/justin/.pulumi/policies/justinvp/pulumi-analyzer-aws-python-v0.0.2.tmp234539259" "/Users/justin/.pulumi/policies/justinvp/pulumi-analyzer-aws-python-v0.0.2"
Creating virtual environment for policy pack...

Finished creating virtual environment

Installing dependencies for policy pack...

Finished installing dependencies

```

We already have https://github.com/pulumi/pulumi-policy/issues/58 tracking improving the experience here, but I'll make a quick improvement while making changes here and change the output to:

```
Installing policy pack aws-python v0.0.2...
Finished installing policy pack
```

The first line would encompass the entire install operation from downloading the policy pack to installing its dependencies. Ideally with https://github.com/pulumi/pulumi-policy/issues/58 we'd add some kind of progress bar, but in lieu of that, the "Installing..." and subsequent "Finished installing" message should suffice.